docs/about-mono/releases/5.10.0.md

```diff
@@ -2,44 +2,161 @@
 layout: releasenotes
 title: 'Mono 5.10.0 Release Notes'
 version: 5.10.0
+releasedate: '2018-02-26'
 ---
 
 Release History
 ===============
 
-TODO
+* 5.10.0.141 - Preview Release _(27 Feb 2018)_
+* 5.10.0.140 - Stable Release _(26 Feb 2018)_
+* 5.10.0.140 - Preview Release _(24 Feb 2018)_
+* 5.10.0.137 - Preview Release _(23 Feb 2018)_
+* 5.10.0.132 - Preview Release _(22 Feb 2018)_
+* 5.10.0.129 - Preview Release _(19 Feb 2018)_
+* 5.10.0.125 - Preview Release _(19 Feb 2018)_
+* 5.10.0.116 - Preview Release _(15 Feb 2018)_
+* 5.10.0.107 - Preview Release _(14 Feb 2018)_
+* 5.10.0.105 - Preview Release _(13 Feb 2018)_
+* 5.10.0.103 - Preview Release _(10 Feb 2018)_
+* 5.10.0.101 - Preview Release _(9 Feb 2018)_
+* 5.10.0.97 - Preview Release _(8 Feb 2018)_
+* 5.10.0.94 - Preview Release _(7 Feb 2018)_
+* 5.10.0.91 - Preview Release _(6 Feb 2018)_
+* 5.10.0.83 - Preview Release _(5 Feb 2018)_
+* 5.10.0.79 - Preview Release _(2 Feb 2018)_
+* 5.10.0.78 - Preview Release _(31 Jan 2018)_
+* 5.10.0.71 - Preview Release _(30 Jan 2018)_
 
 Highlights
 ==========
 
-TODO
+* [Interpreter compiled by default](#interpreter)
+* [Default Interface Method support](#default-interface-method)
+* [Mono SDKs](#mono-sdks)
 
 In Depth
 ========
 
 Runtime
 -------
 
-TODO
+### Interpreter
+
+It is now compiled in by default, and you can use it by running `mono --interpreter`.
+
+We are getting closer to get the interpreter working with JIT/AOT and on mobile:
+
+ - exception handling and debugger support has been unified with JIT/AOT
+ - we support more AppDomain and Remoting cases
+
+It is not yet as feature complete as the JIT, so there will be some behavior differences or breakages; we are currently working on making our complete test suite pass.
+
+### Default Interface Method
+
+The runtime now supports Default Interface Method (https://github.com/dotnet/csharplang/blob/master/proposals/default-interface-methods.md), we now only miss the C# compiler support.
+
+### Mono SDKs
+
+This is a new way of building Mono for some of the platform that we support: Android, iOS and WebAssembly. You can find some documentation at https://github.com/mono/mono/blob/2017-12/sdks/README.md and Makefile build targets at https://github.com/mono/mono/blob/2017-12/sdks/builds.
```

Comment:
`some of the platform` -> platforms

Reply:
I think we should also add some of the goals and why we're doing it, e.g. that Android and iOS would use the logic from Mono instead of building themselves.